[
    {
        "content": "<p>Lots of github CI testing runs seem to be failing at the moment with timeouts in <code>ring_theory.derivation</code> (examples: <a href=\"https://github.com/leanprover-community/mathlib/actions/runs/4198173497\">https://github.com/leanprover-community/mathlib/actions/runs/4198173497</a> or <a href=\"https://github.com/leanprover-community/mathlib/actions/runs/4199207353/attempts/1\">https://github.com/leanprover-community/mathlib/actions/runs/4199207353/attempts/1</a>).</p>",
        "id": 328370841,
        "sender_full_name": "David Loeffler",
        "timestamp": 1676598204
    },
    {
        "content": "<p><a href=\"https://github.com/leanprover-community/mathlib/pull/18457\">#18457</a></p>",
        "id": 328373175,
        "sender_full_name": "Eric Wieser",
        "timestamp": 1676600183
    },
    {
        "content": "<p>I might be responsible for the timeout with my new <a href=\"https://leanprover-community.github.io/mathlib_docs/find/ideal.idem_comm_semiring\">docs#ideal.idem_comm_semiring</a> instance, which makes unification slightly less good in one or two places.</p>",
        "id": 328373373,
        "sender_full_name": "Eric Wieser",
        "timestamp": 1676600383
    },
    {
        "content": "<p><span aria-label=\"ping pong\" class=\"emoji emoji-1f3d3\" role=\"img\" title=\"ping pong\">:ping_pong:</span>, the last bors batch failed due to this timeout</p>",
        "id": 328502915,
        "sender_full_name": "Eric Wieser",
        "timestamp": 1676649075
    },
    {
        "content": "<p><span class=\"user-mention silent\" data-user-id=\"310045\">Eric Wieser</span> has marked this topic as resolved.</p>",
        "id": 328504351,
        "sender_full_name": "Notification Bot",
        "timestamp": 1676649473
    }
]